bool    detected
float32 yaw
float32 pitch
float32 distance
time    stamp
